bool pause
bool terminate_current_drive
---
bool pause_state
bool currently_on_job